sources/meta-yoe/dynamic-layers/qt6-layer/recipes-qt/kiosk-browser/yoe-kiosk-browser.bb

```diff
@@ -66,6 +66,8 @@ DISPLAY_CARD ?= "/dev/dri/card0"
 # Define it via a knob which can be set from config file e.g. local.conf
 YOE_KIOSK_BROWSER_SYSTEMD_UNIT ?= "${@bb.utils.contains('DISTRO_FEATURES', 'wayland', 'yoe-kiosk-browser-wayland.service', 'yoe-kiosk-browser-eglfs.service', d)}"
 
+YOE_KIOSK_BROWSER_XCB = "1"
```

Comment:
doesn;t xcd require X11 protocol, why do we want to enable it, we are intending to use either EGLFS or wayland

Reply:
Hi, this is an option because the latest patch for the yoe-kiosk-browser originates from my project, which uses XCB. Feel free to remove the XCB content if it's not needed.

Reply:
Yes, we want to disable XCB by default -- I'll send another PR.